src/__tests__/dictator-sort.test.ts

```diff
@@ -0,0 +1,54 @@
+import DictatorSort from "@algorithms/dictator-sort";
+const sortedNumbers: readonly number[] = [2, 5, 1234];
+const sortedStrings: string[] = ["list", "of", "strings"];
+
+describe("Dictator Sort Test Suite", () => {
+  let randomOrderNumbers: number[] = [];
+  let randomOrderStrings: string[] = [];
+  beforeAll(() => {
+    randomOrderNumbers = [2, 1, 5, 1234, 58, 21];
+    randomOrderStrings = ["list", "of", "A", "strings"];
+  });
+  it("sorts numbers in asc order by default", () => {
```

Comment:
A better and simpler test would be to see if the returned array and the original array are in the same order since that is what the algo does. You shouldn't have to sort anything